### DISCUSSION TITLE
tileOverlay default on?

### DISCUSSION THREAD

**Discussion by oyvindsolstad** [javalent/obsidian-leaflet#271] [Ideas]
Would it be possible to have a way to set som tileOverlays to be ON by default? 
 
I'm using both overlays from openrailwaymap and openseamap - and for the maps I use them, it's an extra step to turn on those layers EVERY time I go to the note with that map. 

Example: This map of the harbour of Andenes, where I just want the seamarks to be on by default

````leaflet
id: AndenesMAP
lat: 69.325
long: 16.11
height: 500px
tileOverlay:
  - http://tiles.openseamap.org/seamark/{z}/{x}/{y}.png|sjømerker
height: 500px
minZoom: 3
maxZoom: 20
defaultZoom: 14
unit: meters
scale: 1
darkMode: false
````

**Comment by valentine195** [comment]
Hi, this should be in the latest release.

https://github.com/valentine195/obsidian-leaflet-plugin#tile-overlays

**Reply by oyvindsolstad** [reply]
Fantastic! Thank you!

**Comment by ioNetrunner** [comment]
Any chance of getting something like this for imageOverlay too? I use two pictures, a larger map cut in half (for Obsidian Sync file size purposes) where the base image is the right half and the imageOverlay is the left half. Having to turn it on every time I switch pages is getting tiresome. Thanks!
